qiskit_experiments/base_analysis.py

```diff
@@ -87,33 +88,22 @@ def run(
         analysis_options = analysis_options.__dict__
 
         # Run analysis
-        try:
-            analysis_results, figures = self._run_analysis(experiment_data, **analysis_options)
-            for res in analysis_results:
-                if "success" not in res:
-                    res["success"] = True
-        except AnalysisError as ex:
-            analysis_results = [AnalysisResult(success=False, error_message=ex)]
-            figures = None
+        analysis_results, figures = self._run_analysis(experiment_data, **analysis_options)
```

Comment:
Is it OK to remove the try-catch here?  The difficulty is in populating the parameters of `AnalysisResultV1` in case of exception. @jyu00 @nkanazawa1989 @chriseclectic